components/user/UserTabs.js

```diff
@@ -30,6 +31,13 @@ export default function UserTabs({ tabs, setTabs }) {
             <option key={tab.name}>{tab.name}</option>
           ))}
         </select>
+        }
+        {tabs.length==1 && // if tabs has only links in it, then no dropdown is needed
```

Comment:
I don't think this section is required, nothing needs to be shown if only links are available 

Could you add the issue link to the PR description please